### PR TITLE
Final Merge

### PR DESCRIPTION
**Pull request by vjspranav**
A working simulation final merge into main

### CODE REVIEW

**Review by raj-vlabs — CHANGES_REQUESTED** [review]
1. I am not able to see some of the files like aim.md, experiment-name.md, objectives.md, references.md etc. Please ensure all files are present in the repo.
2. Pretest and posttest have sample questions. Please remove
3. .vscode/settings.json need not be checked in. If there's a valid use-case, please let me know.
4. Can we use better names than startAn and unln please?

**Comment by vjspranav** [comment]
> 1. I am not able to see some of the files like aim.md, experiment-name.md, objectives.md, references.md etc. Please ensure all files are present in the repo.
> 2. Pretest and posttest have sample questions. Please remove
> 3. .vscode/settings.json need not be checked in. If there's a valid use-case, please let me know.
> 4. Can we use better names than startAn and unln please?

1. The files mentioned are already present in main branch
2. Added question in pre-test and post-test
3. Removed the file sir
4. Updated function names

File: experiment/simulation/js/main.js [hunk 1]
```diff
@@ -1 +1,83 @@
...
+  tube_ids.forEach((element) => {
+    let path = document.getElementById(element);
+    path.setAttribute("offset", 0);
+  });
+  ids.forEach((ele) => {
+    let path = document.getElementById(ele);
+    path.setAttribute("offset", 0);
+  });
+};
+disablestart = false;
```
Comment (raj-vlabs) [inline experiment/simulation/js/main.js]: Where's this variable declared and where this is used?

File: experiment/simulation/js/main.js [hunk 2]
```diff
@@ -1 +1,83 @@
...
+  return new Promise((resolve) => setTimeout(resolve, ms));
+};
+
+let arrayRemove = (arr, value) => {
+  return arr.filter(function (ele) {
+    return ele != value;
+  });
+};
+
+ids = ["complete-flow"];
```
Comment (raj-vlabs) [inline experiment/simulation/js/main.js]: No declaration for the variable.

**Review by raj-vlabs — CHANGES_REQUESTED** [review]
1. Missing declaration for variable 'ids'.
2. Unused variable 'disablestart'.

**Review by raj-vlabs — APPROVED** [review]

**Pull request merged**